tests/integration/test_design.py

```diff
@@ -1927,3 +1927,111 @@ def test_get_collision(modeler: Modeler):
 
     assert body1.get_collision(body2) == CollisionType.TOUCH
     assert body2.get_collision(body3) == CollisionType.NONE
+
+
+def test_body_scale(modeler: Modeler):
+    """Test for verifying the correct scaling of a ``Body``."""
```

Comment:
```suggestion
    """Verify the correct scaling of a body."""
```

Reply:
a body works @PipKat - thanks for pointing it out nonetheless :)